src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -36,104 +36,64 @@
 #include <chrono>
 #include <thread>
 #include <string>
+#include <utility>
 #include <variant>
 
 #include <rust/ed25519.h>
 
 using namespace libzcash;
 
-int find_output(UniValue obj, int n) {
-    UniValue outputMapValue = find_value(obj, "outputmap");
-    if (!outputMapValue.isArray()) {
-        throw JSONRPCError(RPC_WALLET_ERROR, "Missing outputmap for JoinSplit operation");
-    }
-
-    UniValue outputMap = outputMapValue.get_array();
-    assert(outputMap.size() == ZC_NUM_JS_OUTPUTS);
-    for (size_t i = 0; i < outputMap.size(); i++) {
-        if (outputMap[i].get_int() == n) {
-            return i;
-        }
-    }
-
-    throw std::logic_error("n is not present in outputmap");
-}
-
 AsyncRPCOperation_sendmany::AsyncRPCOperation_sendmany(
```

Comment:
Document the preconditions on arguments.

Reply:
I will also add this documentation to the UA integration branch.